components/places/android/src/main/java/mozilla/appservices/places/PlacesConnection.kt

```diff
@@ -344,7 +344,7 @@ class PlacesWriterConnection internal constructor(conn: UniffiPlacesConnection,
     override fun runMaintenance(dbSizeLimit: UInt) {
         val pruneMetrics = PlacesManagerMetrics.runMaintenanceTime.measure {
             val pruneMetrics = PlacesManagerMetrics.runMaintenancePruneTime.measure {
-                this.conn.runMaintenancePrune(dbSizeLimit)
+                this.conn.runMaintenancePrune(dbSizeLimit, 12U)
```

Comment:
I know we want to make conservative changes here, but I wonder if this specifically can be more aggressive. On. Android in particular, I'm still not sure that this runs often enough to keep up with an active user.

Specifically, maybe doubling (ie, 24) again is ok? I think I'd even be fine with more.

Reply:
I'd be fine with that, but there's also a firefox-android PR to increase the frequency by two.  So if we bumped it to 24 we would be increasing the rate 8x.  Seems fine with me, but I wanted to double check that.